### PR TITLE
for rust context with_context is better, because it is evaluated lazily. 

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**
Fixes: #4129
Signed-off-by: Zhongtao Hu <zhongtaohu.tim@linux.alibaba.com>

### CODE REVIEW

**Review by Tim-Zhang — APPROVED** [review]
Thanks @Tim-0731-Hzt  LGTM

**Review by jodh-intel — APPROVED** [review]
Thanks @Tim-0731-Hzt.

lgtm

File: versions.yaml [hunk 1]
```diff
@@ -279,6 +279,16 @@ languages:
         building Kata
       newest-version: "1.58.1"
 
+  golangci-lint:
```
Comment (wllenyj) [inline versions.yaml:282]: Does this change have anything to do with fixes?

Comment (Tim-0731-Hzt) [reply versions.yaml:282]: No. But it is needed to pass the CI

**Pull request merged**